File: 1944-truncate-sentence/truncate-sentence.java
class Solution {
    public String truncateSentence(String s, int k) {
        String str[]=s.split(" ");
        String res="";
        for(int i=0;i<k;i++){
            if(k-1==i) 
            res+=str[i];
            else
            res+=str[i]+" ";
        }
        return res;
    }
}

File: 1944-truncate-sentence/truncate-sentence.py
class Solution(object):
    def truncateSentence(self, s, k):
        """
        :type s: str
        :type k: int
        :rtype: str
        """
        res=""
        st=s.split(" ")
        for i in range(0,k):
            if i==k-1:
                res+=st[i]
            else :
                res+=st[i]+" "
        return res

        